README.adoc

```diff
@@ -96,6 +96,119 @@ you should use:
 rake preview[staging]
 ----
 
+== Write a blog
+
+Blogs are written in http://asciidoctor.org[Asciidoctor].
+
+Create a file in the directory named after you: for example `posts/Emmanuel`.
+The file should be named as followed: `yyyy-mm-dd-your-lowercase-dash-separated-slug.adoc`.
+For example `2015-06-30-multitenancy-and-current-session.adoc`.
+
+Here is a sample blog template you can start with:
+
+[source]
+.An Asciidoctor blog post
+....
+= Let me tell you about...
+:author: Emmanuel Bernard
+:awestruct-tags: ["Blog", "Test"]
```

Comment:
that's how the Asciidoctor/Awestruct integration is done. Only some tags and auto wired. Otherwise you need to prefix them with `awestruct-`